tests/conftest.py

```diff
@@ -24,6 +25,14 @@ def client():
     yield client
 
 
+# V3 vehicle fixture
+@pytest.fixture(scope="session")
+def v3_vehicle():
+    vehicle_id = os.environ.get("E2E_SMARTCAR_V3_TEST_VEHICLE_ID")
```

Comment:
Can you either add these to the docs somewhere or add default values for the ID and token?
```suggestion
    vehicle_id = os.environ.get("E2E_SMARTCAR_V3_TEST_VEHICLE_ID", "tst2e255-d3c8-4f90-9fec-e6e68b98e9cb")
```